src/modules/stakewise/stakewise-daemon/tasks/run.go

```diff
@@ -83,6 +83,49 @@ func (t *TaskLoop) Run() error {
 		t.wg.Done()
 	}()
 
+	// Initialize send exit data to Nodeset task
```

Comment:
Ah, this isn't really how tasks were designed; there's a lot of duplication here that can be avoided. I'll show you the pattern offline.